roles/confluence/defaults/main.yml

```diff
@@ -1,3 +1,7 @@
 ---
 # defaults file for roles/confluence
-confluence_version: "7.0.4"
+confluence_version: "7.2.0"
```

Comment:
I'm curious if this will upgrade prod when it runs there, or cause any issue.

Reply:
When I started this PR we didn't have a need for an emergency patch. I thought it was have been a slow one. Updated it.